.travis.yml

```diff
@@ -0,0 +1,40 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+language: rust
+rust:
+  - stable
+  - nightly
+matrix:
```

Comment:
When I activate it, nightly fails because of a clippy complaint about [unneeded return in the macro](https://travis-ci.org/maidsafe/system_uri/jobs/184036336#L305). However, if I delete the return (as clippy advises), the macro fails because of type conversion problems (either int or str). I don't even understand why the return statement would be needed or what it does, but it breaks the code to remove it, so I tried to tell clippy to allow the return statement there, but I wasn't able to. It appears to be ignoring the flags set in that file. Not sure what to do to fix that...